<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1061</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>701</width>
      <height>671</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_upload">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>-1</y>
      <width>331</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Загрузка модели из файла</string>
    </property>
    <widget class="QLabel" name="label_filename">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Имя файла:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_filename">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_upload">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>261</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Загрузить модель</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_scale">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>331</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Масштабирование</string>
    </property>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_scale_x">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="minimum">
      <double>-100.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_scale_y">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="minimum">
      <double>-100.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_scale_z">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="minimum">
      <double>-100.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_kx">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>kx</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ky">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ky</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_kz">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>kz</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_scale">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>261</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="text">
      <string>Масштабировать</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_move">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>331</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Перенос</string>
    </property>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_move_x">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="minimum">
      <double>-100.000000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_move_y">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="minimum">
      <double>-100.000000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_move_z">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="minimum">
      <double>-100.000000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_dx">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>dx</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_dy">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>dy</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_dz">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>dz</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_move">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>261</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Перенести</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_rotate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>331</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Поворот</string>
    </property>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_rotate_x">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="minimum">
      <double>-360.000000000000000</double>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_rotate_y">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="minimum">
      <double>-360.000000000000000</double>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_rotate_z">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="minimum">
      <double>-360.000000000000000</double>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_ax">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ax</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ay">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ay</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_az">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>az</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_rotate">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>261</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Повернуть</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_return_default">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>690</y>
      <width>321</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Вернуть начальное положение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save_current">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>690</y>
      <width>321</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1061</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
